vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -67,19 +88,48 @@ int main(int argc, char** argv)
         return l_rc;
     }
 
-    if (*l_hardwareFlag && !std::filesystem::exists(l_vpdPath))
-    {
-        std::cerr << "Given EEPROM file path doesn't exist : " + l_vpdPath
-                  << std::endl;
-        return l_rc;
-    }
-
-    (void)l_fileOption;
-
     if (*l_readFlag)
     {
         // TODO: call read keyword implementation from here.
     }
+    else if (l_writeFlag->count() > 0)
+    {
+        if ((l_hardwareFlag->count() > 0) &&
```

Comment:
if(l_hardwareFlag->count() == 1).
similarly check for count == 1 in other places as well. Do not allow the user to execute the tool if the user given command is incorrect

Reply:
my comment is same as above.